src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/ConfigurationSerializer.cs

```diff
@@ -43,6 +44,7 @@ public MetricsDeclarationV1 InterpretYamlStream(YamlMappingNode rootNode)
 
             var metricsDeclaration = new MetricsDeclarationV1
             {
+                Version = SpecVersion.v1.ToString(),
```

Comment:
Would it make sense to just move this to `MetricsDeclarationV1` itself?

Reply:
Sure.